Examples/FriendlyFlix/app/FriendlyFlix/FriendlyFlix/App/FriendlyFlixApp.swift

```diff
@@ -0,0 +1,60 @@
+// Copyright 2024 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import Firebase
+import FirebaseAuth
+import FirebaseDataConnect
+import FriendlyFlixSDK
+import SwiftUI
+
+@main
+struct FriendlyFlixApp: App {
+  private func loadRocketSimConnect() {
+    #if DEBUG
+      guard Bundle(
+        path: "/Applications/RocketSim.app/Contents/Frameworks/RocketSimConnectLinker.nocache.framework"
+      )?
+        .load() == true else {
+        print("Failed to load linker framework")
+        return
+      }
+      print("RocketSim Connect successfully linked")
+    #endif
+  }
+
+  var authenticationService: AuthenticationService?
+
+  init() {
+    loadRocketSimConnect()
+    FirebaseApp.configure()
+
```

Comment:
If the sample is referring to the emulator, you may need this line here

```
DataConnect.friendlyFlixConnector.useEmulator(port: 9399)
```

Reply:
Should we use a launch argument to switch environments?

Reply:
Added a launch argument to make it easier to switch between Emulator and cloud project.